test/presubmit-config-check.sh

```diff
@@ -27,19 +34,19 @@ export TESTGRID_CONFIG=$(mktemp)
 subheader "Regenerating config for production prow and testgrid"
 make -C config/prow config
 subheader "Comparing the generated config files with the existing config files"
-diff --ignore-matching-lines="^# Copyright " "${PROW_CONFIG}" "config/prow/core/config.yaml"
-diff --ignore-matching-lines="^# Copyright " "${PROW_JOB_CONFIG}" "config/prow/jobs/config.yaml"
-diff --ignore-matching-lines="^# Copyright " "${PROW_PLUGINS}" "config/prow/core/plugins.yaml"
-diff --ignore-matching-lines="^# Copyright " "${TESTGRID_CONFIG}" "config/prow/testgrid/testgrid.yaml"
+diff_config_files "${PROW_CONFIG}" "config/prow/core/config.yaml"
+diff_config_files "${PROW_JOB_CONFIG}" "config/prow/jobs/config.yaml"
+diff_config_files "${PROW_PLUGINS}" "config/prow/core/plugins.yaml"
```

Comment:
I meant the set of these three should be in a function

Reply:
Done.